visualize-admin/visualization-tool
# feat: Show tree in MultiFilters

Closes #800.

[Design specs](https://www.notion.so/interactivethings/V04-Indent-and-new-UI-844455727a2b421ca00e0309c3a25957)

- Filter values in drawer are displayed as tree
- Search in the tree keeps tree structure (e.g. when a child node is "found", all of its parents will also be displayed)

### How to test

Look for a dataset with hierarchies (e.g. Bathing water quality (with Hierarchie) or Öffentliche Ausgaben TEST 3) and without hierarchies (e.g. Bathing water quality) and create a chart with color segmentation; afterwards edit filters to see if it works correctly.
